DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -40,7 +61,26 @@ public void CleanDb()
                 dbContext.Database.EnsureDeleted();
             }
         }
+      
+        #region EditPosition
+        [Test]
+        public void SuccessfullyEditPosition()
+        {
+            newPosition = new DbPosition
+            {
+                Id = positionId,
+                Name = "abracadabra",
+                Description = "bluhbluh"
+            };
 
+            var result = repository.EditPosition(newPosition);
+            DbPosition newDbPosition = dbContext.Positions.FirstOrDefault(dbPosition => dbPosition.Id == positionId);
```

Comment:
Change name(not necessary). Maybe, newDbPosition to updatedPosition

Reply:
Fixed